backend/backend/src/main/java/de/aaaaaaah/velcom/backend/listener/github/GithubPrInteractor.java

```diff
@@ -571,7 +571,21 @@ private String buildFinishedPrReply(FinishedGithubCommand reply, RunId runId,
 			.append(".");
 
 		if (reasons == null) {
-			builder.append("\nThere were no significant changes.");
+			if (compareToRuns.size() == 1) {  // no idea why we would even have more than 1
```

Comment:
When trying to compare commit `I` against branch `branch` in a scenario like the following:
```
A - B - C - D - E - branch
     \       \
      F - G - H - I
```
compareToRuns would include both B and D.

Reply:
Seems like a stretch to assume that anyone would ever be interested in `B` :sweat_smile: . Anyway, not an issue for Lean.

Reply:
One can easily think up scenarios (though not for Lean's linear structure) where the two or more commits in question are symmetric:
```
      C - D ----.
     /     \     \
A - B       \     G - branch
     \       \   /
      E - F --\-´
           \   H - I
            `-´
```
Comparing `I` against `branch` here would include `D` and `F` in `compareToRuns`.